src/transformers/generation_tf_utils.py

```diff
@@ -16,6 +16,7 @@
 
 import inspect
 from dataclasses import dataclass
+from functools import wraps
```

Comment:
Nit: I'd prefer to just import `functools` and then call `functools.wraps` later on to make it clearer where the `wraps` comes from.